### PR TITLE
use only one function for plotting

### PR DESCRIPTION
**Pull request by rodriguez-facundo**
this requires: 
https://github.com/MetaCell/NetPyNE-UI/pull/69
https://github.com/Neurosim-lab/netpyne/pull/353

### CODE REVIEW

**Comment by rodriguez-facundo** [comment]
@adrianq now is ready for merging =)

**Comment by adrianq** [comment]
@rodriguez-facundo Thanks for this. A nice refactor that removes a lot of redundant code. I just added a mechanism to display the error if any in a dialog and some minor refactors. It works smoothly except for LFP Plots where I get this error:
screenshot from 2018-08-26 00-06-32

Is this normal? I am using tut3 and I added a LFP Plot in the first tab.

File: components/instantiation/NetPyNEInstantiated.js [hunk 1]
```diff
@@ -78,18 +78,26 @@ export default class NetPyNEInstantiated extends React.Component {
...
+                dialogTitle: "NetPyNE returned an error plotting " + plotName,
+                dialogMessage: parsedResponse['message'] + "\n " + parsedResponse['details'],
+                openDialog: true
+            });
+            return true;
+        }
+        return false;
+    }
+
+    plotFigure(plotName, plotMethod, flavour=false) {
```
Comment (salvadord) [inline components/instantiation/NetPyNEInstantiated.js]: maybe call it `plotType` or similar instead of `flavour` (with the British spelling!)

Comment (rodriguez-facundo) [reply components/instantiation/NetPyNEInstantiated.js]: I'm learning new spellings just in case something happens in the future.

**Pull request merged**